### ISSUE TITLE
Could not find intellij-core.jar (com.android.tools.external.com-intellij:intellij-core:26.0.1)

### ISSUE THREAD

**Issue by skymax406** [pusherman/react-native-network-info#65]
Hello i have this issue with ./gradlew clean command

`A problem occurred configuring project ':react-native-network-info'.
> Could not resolve all artifacts for configuration ':react-native-network-info:classpath'.
   > Could not find intellij-core.jar (com.android.tools.external.com-intellij:intellij-core:26.0.1).
     Searched in the following locations:
         https://jcenter.bintray.com/com/android/tools/external/com-intellij/intellij-core/26.0.1/intellij-core-26.0.1.jar
`

An idea ?

**Comment by tekmin** [comment]
the solution is to put `google()` before `jcenter()` in project/build.gradle https://stackoverflow.com/a/52952806

**Comment by skymax406** [comment]
Already do that, it's not working for me. I have test on another computer, cleaning gradle cache, ect ... no change

**Comment by skymax406** [comment]
@tekmin However, i've test to put `google()` before `jcenter() `in project/build.gradle but of react-native-network-info project, not of MY project, and it's working. Maybe a correction on the lib ?

**Comment by martinffx** [comment]
I'm experiencing the exact same issue, putting the `jcenter()` before `google()` in `android/build.gradle` doesn't resolve the issue. But if I do the edit in `node_modules/react-native-network-info/android/build.gradle` the issue is resolved.

**Comment by martinffx** [comment]
Submitted PR #66

**Comment by tekmin** [comment]
@skymax406 yes it's should be the react-native-network-info project gradle as @martinffx PR
